addons/account/static/src/css/account_payment.css

```diff
@@ -0,0 +1,27 @@
+.o_popover_header {
+    padding: 5px 0 5px 8px;
+    border-bottom: 1px solid #ccc;
+}
+
+.o_memo_content {
+    max-width: 200px;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    display: -webkit-box;
+    -webkit-line-clamp: 3;
```

Comment:
I'm really not a big fan of this property. I think it will be better to just set a number of character maximum in the template and don't use css for that.

Reply:
I think using this property for specifying the number of lines is better than setting the max number of chars, as setting the number of chars doesn't necessarily correspond to exactly 3 lines and it's hard to specify the exact number of chars needed.